dev/jsx/weather.jsx

```diff
@@ -7,6 +7,10 @@ import SearchBar from './search-bar';
 import WeatherCard from './weather-card';
 import Geosuggest from 'react-geosuggest';
 
+const API_KEY = '5332856fca0fe1e7';
```

Comment:
You may want to create a `const.js` file that you import that has API key, URLs, etc for your app.